### ISSUE TITLE
Watch2Gether / w2g.tv is not working anymore (Last.fm)

### ISSUE THREAD

**Issue by mah0ne1** [web-scrobbler/web-scrobbler#4724]
**Describe the bug**
The scrobbling on Last.fm is not working anymore for Watch2Gether / w2g.tv. While playback is ongoing, the extension shows "Play some music to get started."

**Environment (please complete the following information)**
-   OS: Windows 10 Pro
-   Browser: Google Chrome 124.0.6367.210 
-   Extension version: 3.7.0

**Additional context**
I believe it isn't related to my setup at all, I would assume the "_connector code for this specific website needs to be updated_," as is stated by the extension itself in the non-scrobbling state.

**Comment by coqre** [comment]
same to me :(

**Comment by rob9315** [comment]
yes, this is broken. marking as a non-actual duplicate of the youtube embed issue
